tests/helper/helper_oc.go

```diff
@@ -577,7 +577,7 @@ func (oc OcRunner) GetNamespaceProject() string {
 }
 
 func (oc OcRunner) HasNamespaceProject(name string) bool {
-	out := Cmd(oc.path, "get", "project", name, "-o", "jsonpath={.metadata.name}").
+	out := Cmd(oc.path, "get", "project", "-o", "jsonpath={.metadata.name}").
```

Comment:
This is not going to work, as you removed the `name` parameter the command will return all project on the cluster so the `jsonpath` is no longer valid as not the command returns the array